packages/forms/src/Components/Concerns/HasHint.php

```diff
@@ -34,18 +34,34 @@ protected function setUpHint(): void
         $this->afterLabel(function (Field | Placeholder $component): array {
             $components = [];
 
-            $hint = $component->getHint();
-
-            if (filled($hint)) {
-                $components[] = Text::make($hint)
-                    ->color($component->getHintColor());
+            if ($component->hasHint()) {
+                $components[] = Text::make(static function (Text $component) {
+                    /** @phpstan-ignore-next-line */
+                    return $component->getContainer()->getParentComponent()->getHint();
```

Comment:
The PHPStan was complaining after that it didn't understand the return type of `getParentComponent()` as having the `getHint()` method (etc).